### ISSUE TITLE
[API][WEBAPP] Lorsqu'on clique sur un résultat de recherche, amener sur un carousel des résultats de la recherche

### ISSUE THREAD

**Issue by arnoo** [betagouv/pass-culture-browser#850]

**Comment by IonLzr** [comment]
@arnoo On avait dit que clic sur _un_ résultat = arrivée sur _la_ fiche détail de l'offre. 
Celle-ci contient un bouton back pour revenir aux résultats de recherche, mais rien d'autre. Pour rester dans la section recherche de l'app.
Si tu veux revenir au carrousel, tu fermes la Recherche par le bouton X dans le header.
On fait ça ?

**Comment by arnoo** [comment]
Ah ok, comme tu veux, c'est juste qu'aujourd'hui tu as accès au carousel depuis la fiche offre et que si tu fais défiler ça n'a rien a voir avec ta recherche, ce qui est bizarre.

**Comment by IonLzr** [comment]
Oui, je me doutais bien que c'était la raison. Ce parcours est pas simple… du coup, je me dis qu'il vaut mieux le simplifier quitte à couper des bouts : 
- on vient de la recherche = on est plus en mode découverte = on a moins besoin de l'accroche plein écran 
—> ce serait bien de **bloquer l'accès** au mode carrousel depuis la fiche détail ;
—> depuis cette fiche détail, on ne peut **que retourner** à la recherche ;

Le carrousel reste ainsi réservé à la découverte et à nos recos.
